node-proxy/openshift-origin-node-proxy.spec

```diff
@@ -132,6 +132,7 @@ fi
 %endif
 %attr(0755,-,-) %{_bindir}/node-find-proxy-route-files
 %attr(0640,-,-) %{_sysconfdir}/openshift/web-proxy-config.json
+%config(noreplace) %{_sysconfdir}/openshift/web-proxy-config.json
```

Comment:
Do we know whether upgrading from a version without `%config(noreplace)` to a version with it will create the `.rpmnew` file or just overwrite the existing file? Mostly I'm just curious; it doesn't matter as much now that a default value is specified in the code.